RELEASE_NOTES.md

```diff
@@ -4,26 +4,135 @@
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/IncludingXMLRubyRecipe
 
+The `IncludingXMLRubyRecipe` detects cookbooks that include the `xml::ruby` recipe. This recipe was used to install the `nokogiri` gem into Chef Infra Client for use in parsing XML in cookbooks. Nokogori ships with Chef Infra Client 13+, so this recipe is no longer necessary.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/LegacyYumCookbookRecipes
 
+The `LegacyYumCookbookRecipes` cop detects usage of legacy [yum](https://supermarket.chef.io/cookbooks/yum) cookbook recipes. These recipes were removed from the yum cookbook in verson 3.0, which was released in 2013. Cookbooks should be updated to use the new individual yum cookbooks that contain the same functionality.
+
+
+`Examples`
+
+Legacy recipes:
+```ruby
+  include_recipe 'yum::epel'
+  include_recipe 'yum::ius'
+```
+
+New cookbooks containing the same functionality:
+```ruby
+  include_recipe 'yum-epel'
+  include_recipe 'yum-ius'
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/DefaultMetadataMaintainer
 
+The `DefaultMetadataMaintainer` cop detects the default `maintainer` and `maintainer_email` fields generated by the `chef generate cookbook` command. Cookbook metadata should be updated with actual maintainer information.
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/UsesDeprecatedMixins
 
+The `UsesDeprecatedMixins` cop detects various legacy mixins used in `HWRP` and `LWRP` resources. These mixins are no longer necessary and were removed in Chef Infra Client 14, which will cause errors running these resources on modern Chef Infra Client releases.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/MinitestHandlerUsage
 
+The `MinitestHandlerUsage` cop detects cookbooks that depend on the [minitest-handler](https://supermarket.chef.io/cookbooks/minitest-handler) cookbook. This cookbook provides a handler for running [minitests](https://github.com/seattlerb/minitest) after each Chef Infra Client run. The handler is no longer maintained and we highly recommend using InSpec for validating system configuration post Chef Infra Client run. InSpec includes over 100 build in resources and integrates with Automate to report the results of system state validation.
```

Comment:
```suggestion
The `MinitestHandlerUsage` cop detects cookbooks that depend on the [minitest-handler](https://supermarket.chef.io/cookbooks/minitest-handler) cookbook. This cookbook provides a handler for running [minitests](https://github.com/seattlerb/minitest) after each Chef Infra Client run. The handler is no longer maintained and we highly recommend using InSpec for validating system configuration post-Chef Infra Client run. InSpec includes over 100 builds in resources and integrates with Automate to report the results of system state validation.
```

Reply:
Please check if the phrasing of "100 builds in resources" in the last sentence is the intended phrasing.

Reply:
🤦‍♂ nope

Reply:
Fixed